pages/index.tsx

```diff
@@ -1,71 +1,51 @@
-import Head from 'next/head'
-import Image from 'next/image'
-import styles from '../styles/Home.module.css'
+import Head from "next/head";
+import Header from "../components/Header";
+import Footer from "../components/Footer";
+import Card from "../components/Card";
+
+import styles from "../styles/Home.module.css";
+import styled from "styled-components";
+
+const dummyArray = [
+  {
+    name: "Walter",
+  },
+  {
+    name: "Eshi",
+  },
+  {
+    name: "Andy",
+  },
+  {
+    name: "Elgin",
+  },
+  {
+    name: "Kirsten",
+  },
+  {
+    name: "Neele",
+  },
+  {
+    name: "Waltertest",
+  },
+];
 
 export default function Home() {
   return (
     <div className={styles.container}>
       <Head>
-        <title>Create Next App</title>
+        <title>MatchBall</title>
         <meta name="description" content="Generated by create next app" />
         <link rel="icon" href="/favicon.ico" />
       </Head>
-
+      <Header />
       <main className={styles.main}>
-        <h1 className={styles.title}>
-          Welcome to <a href="https://nextjs.org">Next.js!</a>
-        </h1>
-
-        <p className={styles.description}>
-          Get started by editing{' '}
-          <code className={styles.code}>pages/index.tsx</code>
-        </p>
-
-        <div className={styles.grid}>
-          <a href="https://nextjs.org/docs" className={styles.card}>
-            <h2>Documentation &rarr;</h2>
-            <p>Find in-depth information about Next.js features and API.</p>
-          </a>
-
-          <a href="https://nextjs.org/learn" className={styles.card}>
-            <h2>Learn &rarr;</h2>
-            <p>Learn about Next.js in an interactive course with quizzes!</p>
-          </a>
-
-          <a
-            href="https://github.com/vercel/next.js/tree/canary/examples"
-            className={styles.card}
-          >
-            <h2>Examples &rarr;</h2>
-            <p>Discover and deploy boilerplate example Next.js projects.</p>
-          </a>
-
-          <a
-            href="https://vercel.com/new?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
-            target="_blank"
-            rel="noopener noreferrer"
-            className={styles.card}
-          >
-            <h2>Deploy &rarr;</h2>
-            <p>
-              Instantly deploy your Next.js site to a public URL with Vercel.
-            </p>
-          </a>
+        <div className={styles.cardContainer}>
+          {dummyArray.map(({ name }) => {
+            return <Card key={name} name={name} />;
```

Comment:
If you can use your own data, please provide an id. You will need it for the key (more robust) and also for anything you do in future with the data.